hydra-node/test/Hydra/Chain/Direct/StateSpec.hs

```diff
@@ -398,7 +398,7 @@ forAllFanout action =
        in action utxo tx
             & label ("Fanout size: " <> prettyLength (countAssets $ txOuts' tx))
  where
-  maxSupported = 45
+  maxSupported = 39
```

Comment:
This should not be impacted much. The only difference is the slight increase of head datum size due to the contestation period. Try to not lower it too much.

Reply:
I tried higher values and they all fail.

unfortunately, this is the lowest value it supports.